integration/commands_administration_test.go

```diff
@@ -138,8 +138,10 @@ func TestCommandsAdministrationCreateDropListDatabases(t *testing.T) {
 	assert.Equal(t, bson.D{{"ok", 1.0}}, res)
 }
 
-func TestCommandsAdministrationListDatabases(t *testing.T) {
-	t.Parallel()
+func TestCommandsAdministrationListDatabases(tt *testing.T) {
```

Comment:
Could you check why below tests are failing on CI for sqlite please :pray: 
`TestCommandsAdministrationCreateDropListDatabases`
`TestCommandsAdministrationServerStatusMetrics`
`TestCommandsAdministrationCompatCollStatsWithScale`
`TestCommandsAdministrationKillCursors`
`TestCommandsAdministrationListCollections`
`TestCommandsAdministrationBuildInfo`
`TestCommandsAdministrationCompatDBStatsWithScale`
`TestCommandsDiagnosticHostInfo`
`TestCommandsAdministrationDataSizeErrors`
`TestCommandsAuthenticationLogout`
`TestCommandsDiagnosticConnectionStatus`
`TestCommandsAdministrationCollStatsEmpty`
```


They fail like this
```
2023-08-02T15:45:16.9055686Z --- FAIL: TestCommandsAdministrationListCollections (0.15s)
2023-08-02T15:45:16.9055834Z     commands_administration_test.go:172: 
2023-08-02T15:45:16.9056147Z         	Error Trace:	/home/runner/work/FerretDB/FerretDB/integration/setup/listener.go:152
2023-08-02T15:45:16.9056721Z         	            				/home/runner/work/FerretDB/FerretDB/integration/setup/setup_compat.go:95
2023-08-02T15:45:16.9057300Z         	            				/home/runner/work/FerretDB/FerretDB/integration/setup/setup_compat.go:122
2023-08-02T15:45:16.9057935Z         	            				/home/runner/work/FerretDB/FerretDB/integration/commands_administration_test.go:172
2023-08-02T15:45:16.9058109Z         	Error:      	Received unexpected error:
2023-08-02T15:45:16.9059032Z         	            	[registry.go:76 metadata.NewRegistry] [registry.go:92 metadata.(*Registry).loadCollections] SQL logic error: no such table: _ferretdb_collections (1)
2023-08-02T15:45:16.9059271Z         	Test:       	TestCommandsAdministrationListCollections
```